app/main/spawn.js

```diff
@@ -61,7 +61,7 @@ export function startUserAgentService(userAgentClient, options = {}) {
   // Careful passing in options.attached as true, as this stops the service from
   // outliving the its original parent, which we want to allow in the UAS case.
   const detached = !options.attached; // Detach by default
-  const stdio = detached ? ['ignore'] : ['ignore', process.stdout, process.stderr];
+  const stdio = ['ignore', process.stdout, process.stderr];
```

Comment:
So this will force the UA service to exit when the main process does. Is that what we want?